projects/ad_fmclidar1_ebz/README.md

```diff
@@ -0,0 +1,115 @@
+
+![HDL Block Diagram](./doc/img/hdl_lidar.png)
+
+### AXI_LASER_DRIVER IP
+
+The axi_laser_driver IP is responsible to generate a narrow pulse for the laser
+driver circuit, to control the TIA channel selection on the analog front end (AFE)
+board, and to synchronize the data acquisition to the generated pulses.
+
+More information about the IP can be found [here](https://wiki.analog.com/resources/fpga/docs/axi_laser_driver).
+
+### Control interfaces
+
+| Name | Type | Details |
+| ---- | ---- | ------- |
+| adc_fd*                     | GPIO        | Monitors the AD9094 Fast detect output lines | 
+| adc_pwdn                    | GPIO        | Controls the AD9094 Power-Down input line | 
+| spi_adc_*                   | 4-wire SPI  | AD9094 configuration interface | 
+| spi_vco_*                   | 3-wire SPI  | ADF436-7 configuration interface | 
+| spi_clkgen_*                | 4-wire SPI  | AD9528 configuration interface | 
+| laser_driver_p\n            | LVDS output | It controls the laser driver circuit, it is generated by the axi_laser_driver IP instance | 
+| laser_gpio[13:0]            | GPIO        | Unused GPIO line on the lase board |
+| tia_chsel[7:0]              | CMOS output | TIA channel selection lines, it is controlled by the axi_laser_driver instance |
+| afe_dac_sda\scl\load\clr_n  | I2C/GPIO    | AD5627 configuration interface |
+| rx_ref_clk_p\n              | LVDS        | JESD204B reference clock for the high-speed gigabit transceivers; runs at 250MHz |
+| rx_device_clk_p\n           | LVDS        | JESD204B device clock for the transport layer and additional data processing; runs at 250MHz |
+| rx_data_p\n[3:0]            | CML         | JESD204B high-speed serial lanes; runs at 10Gbps |
+| rx_sync_p\n[1:0]            | LVDS        | JESD204B SYNC signals for interface synchronization |
+| rx_sysref_p\n               | LVDS        | JESD204B SYSREF signal for deterministic latency |
+
+### JESD204B interface
+
+The JESD204B interface runs in Subclass 1 mode to ensure the deterministic latency 
+of the link. The following tables are summarizing the JESD204B important configuration 
+parameter and attributes.
+
+| Parameter name | Abbreviation |Value |
+| -------------- | ------------ | ---- |
+| Number of lanes  | L | 4 | 
+| Number of converter | M | 4 |
+| Converter resolution | NP | 8 |
+| Total number of Bits per Sample| NP | 8 |
+| Samples per frame | S | 1 |
+| Octets per frame | F | 1 |
+| Frames per Multiframe | K | 32 |
+| Number of control bits | CS | 0 |
+
+| Rates and Clocks | Value | 
+| ---------------- | ----- |
+| Sample rate | 1GSPS |
+| Lane rate | 10Gbps |
+| GT reference clock | 250MHz |
+| Device clock | 250 MHz | 
+
+## References
+
++ AD-FMCLIDAR1-EBZ product page: https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/AD-FMCLIDAR1-EBZ.html
++ AD-FMCLIDAR1-EBZ user guide: https://wiki.analog.com/resources/eval/user-guide/ad-fmclidar1-ebz
```

Comment:
The correct link is: https://wiki.analog.com/resources/eval/user-guides/ad-fmclidar1-ebz